pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -228,6 +228,21 @@ stdenv.mkDerivation (finalAttrs: {
 
   enableParallelBuilding = true;
 
+  # For the same reason we don't split "lib" output we undo the $target/
+  # prefix for installed headers and libraries we link:
+  #   $out/$host/$target/lib/*     to $out/lib/
+  #   $out/$host/$target/include/* to $dev/include/*
+  # TODO(trofi): fix installation paths upstream so we could remove this
+  # code and have "lib" output unconditionally.
+  postInstall = lib.optionalString (hostPlatform != targetPlatform) ''
+    for e in $out/${hostPlatform.config}/${targetPlatform.config}/lib/*; do
+      ln -vs $e $out/lib/
+    done
+    for e in $out/${hostPlatform.config}/${targetPlatform.config}/include/*; do
+      ln -vs $e $dev/include/
```

Comment:
I think you could inline the globs here and avoid the loop and many `ln`s (unless there's a huge number of headers).

Reply:
Ah, good point! Yeah, it's just 7 files. Will do in a few hours.

Reply:
Updated to avoid `for` loops and using globs directly.